duckduckgo/content-scope-scripts
# ntp: favorites drag and drop

**Asana Task/Github Issue:** https://app.asana.com/0/0/1208684961069939/f

## Description

- [x] Added drag+drop for favorites
- [x] Tested on Windows during development
- [ ] Will be subject to security review before deployed to production 

## Testing Steps

- visit https://deploy-preview-1150--content-scope-scripts.netlify.app/build/pages/new-tab/
- ensure you can drag and drop items
- ensure you can toggle the section entirely

## Checklist


*Please tick all that apply:*
